quickstarts/router-ext/README.md

```diff
@@ -0,0 +1,258 @@
+Next, build the custom Smart Router image.
+
+Create a file called Dockerfile with the following content:
+
+```bash
+from registry.redhat.io/rhpam-7/rhpam-smartrouter-rhel8:7.9.0
+
+RUN rm -rfv /opt/rhpam-smartrouter/openshift-launch.sh
+COPY openshift-launch.sh  /opt/rhpam-smartrouter/openshift-launch.sh
+
+COPY router-ext-0.0.1-SNAPSHOT.jar /opt/rhpam-smartrouter/router-ext-0.0.1-SNAPSHOT.jar
+
+USER root
+RUN chown jboss. /opt/rhpam-smartrouter/router-ext-0.0.1-SNAPSHOT.jar /opt/rhpam-smartrouter/openshift-launch.sh
+RUN chmod +x /opt/rhpam-smartrouter/openshift-launch.sh
+
+# run image with jboss user.
+USER 185
+```
+
+Now, build the image:
+
+```bash
+$ docker build -t my-smartrouter-ext:1.0 .
+docker build -t my-smartrouter-ext:1.0 .
+Sending build context to Docker daemon   12.8kB
+Step 1/8 : from rhpam-7/rhpam-smartrouter-rhel8:7.8.1
+ ---> 1822bf9d5565
+Step 2/8 : RUN rm -rfv /opt/rhpam-smartrouter/openshift-launch.sh
+ ---> Using cache
+ ---> f9bcb5f6581f
+Step 3/8 : COPY openshift-launch.sh  /opt/rhpam-smartrouter/openshift-launch.sh
+ ---> Using cache
+ ---> bb9cd6c473db
+Step 4/8 : COPY router-ext-0.0.1-SNAPSHOT.jar /opt/rhpam-smartrouter/router-ext-0.0.1-SNAPSHOT.jar
+ ---> 490e27e594a0
+Step 5/8 : USER root
+ ---> Running in 70ad33a793ee
+Removing intermediate container 70ad33a793ee
+ ---> eb73b95c8e51
+Step 6/8 : RUN chown jboss. /opt/rhpam-smartrouter/router-ext-0.0.1-SNAPSHOT.jar /opt/rhpam-smartrouter/openshift-launch.sh
+ ---> Running in 78597febac3b
+Removing intermediate container 78597febac3b
+ ---> 3b1678c0def8
+Step 7/8 : RUN chmod +x /opt/rhpam-smartrouter/openshift-launch.sh
+ ---> Running in a13e36a57a50
+Removing intermediate container a13e36a57a50
+ ---> e5829697cb44
+Step 8/8 : USER 185
+ ---> Running in 5639fe7e585f
+Removing intermediate container 5639fe7e585f
+ ---> 18a546161039
+Successfully built 18a546161039
+Successfully tagged my-smartrouter-ext:1.0
+
+```
+
+To verify the built image, check that the `openshift-launch.sh` script was correctly updated:
+
+```bash
+$ docker run --rm my-smartrouter-ext:1.0 cat /opt/rhpam-smartrouter/openshift-launch.sh
+...
+exec ${JAVA_HOME}/bin/java ${SHOW_JVM_SETTINGS} "${D_ARR[@]}" \
+    -cp /opt/${JBOSS_PRODUCT}/router-ext-0.0.1-SNAPSHOT.jar:/opt/${JBOSS_PRODUCT}/${KIE_ROUTER_DISTRIBUTION_JAR} \
+    org.kie.server.router.KieServerRouter
+```
+
+If the output is the same, you are ready to proceed and run the image to make sure it is working:
+
+```bash
+...
+INFO: Using 'LatestVersionContainerResolver' container resolver and restriction policy 'ByPassUserNotAllowedRestrictionPolicy'
+...
+INFO: KieServerRouter started on :9000 at Tue Sep 01 20:23:08 UTC 2020
+```
+
+`LatestVersionContainerResolver` is the custom container resolver that the extension adds to the 
+Smart Router image.
+
+
+## Deploying the custom image on OpenShift using RHPAM Operator.
+
+For this tutorial, I'll be using the CodeReady Containers.
```

Comment:
```suggestion
For this tutorial, we use CodeReady Containers.
```

Reply:
same here, it is a personal choice, "we use" sounds a bit strange in this context.

Reply:
All right - it's not what could fly in docs - but this is github so let it be